src/cobra/summary/metabolite_summary.py

```diff
@@ -0,0 +1,399 @@
+"""Provide the metabolite summary class."""
+
+
+import logging
+from operator import attrgetter
+from textwrap import shorten
+from typing import TYPE_CHECKING, List, Optional, Union
+
+from pandas import DataFrame
+
+from cobra.flux_analysis import flux_variability_analysis, pfba
+from cobra.summary import Summary
+
+
+if TYPE_CHECKING:
+    from cobra.core import Metabolite, Model, Reaction, Solution
+
+
+logger = logging.getLogger(__name__)
+
+
+class MetaboliteSummary(Summary):
+    """
+    Define the metabolite summary.
+
+    Attributes
+    ----------
+    producing_flux : pandas.DataFrame
+        A table of only the producing fluxes.
+    consuming_flux : pandas.DataFrame
+        A table of only the consuming fluxes.
+
+    See Also
+    --------
+    Summary : Parent that defines further attributes.
+    ReactionSummary
+    ModelSummary
+
+    """
+
+    def __init__(
+        self,
+        *,
+        metabolite: "Metabolite",
+        model: "Model",
+        solution: Optional["Solution"] = None,
+        fva: Optional[Union[float, "DataFrame"]] = None,
```

Comment:
Since, we are importing `pandas` at runtime, can we use `pd.DataFrame` or `DataFrame` instead of `"DataFrame"`?